deploy/clowdapp.yaml

```diff
@@ -122,6 +122,8 @@ objects:
             value: ${PROMETHEUS}
           - name: CONSOLEDOT_BASE_URL
             value: ${CONSOLEDOT_BASE_URL}
+          - name: ENVIRONMENT_NAME
```

Comment:
We shouldn't need this, we already passing in the console url in the `base_console_url` slot.

Reply:
Ah, actually, we are doing something different with this. The sso url is actually different from the base console dot. We send this environment name (`stage|prod`) to the frontend and it chooses which sso url to use. [Here](https://github.com/RedHatInsights/astro-virtual-assistant-frontend/blob/main/src/api/PostManage2fa.ts#L34)